### ISSUE TITLE
concurrent.futures._base.TimeoutError after 2 push_updates

### ISSUE THREAD

**Issue by nriley** [postlund/pyatv#68]
```
% atvremote --login_id 0x0000000000000001 --address bedroom.local push_updates
Press ENTER to stop
Media type: Music
Play state: Playing
     Title: Aaron Severson of AteUpWithMotor.com
    Artist: Petersen Automotive Museum
     Album: CarStories
  Position: 378/3003s (12.6%)
    Repeat: Off
   Shuffle: False
--------------------
Media type: Music
Play state: Playing
     Title: Aaron Severson of AteUpWithMotor.com
    Artist: Petersen Automotive Museum
     Album: CarStories
  Position: 382/3004s (12.7%)
    Repeat: Off
   Shuffle: False
--------------------
Traceback (most recent call last):
  File "/Users/nicholas/Documents/Development/pyatv/src/pyatv/pyatv/__main__.py", line 306, in _run_application
    return (yield from asyncio.wait_for(cli_handler(loop), timeout=15))
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/asyncio/tasks.py", line 356, in wait_for
    raise futures.TimeoutError()
concurrent.futures._base.TimeoutError
```

This happened after the 1st update on Python 3.5; I upgraded to Python 3.6 and it now gets 2 updates before no longer working.  This is a 3rd gen Apple TV to which I am sending AirPlay audio (there's actually nothing plugged into the HDMI port) if that is of any importance.

**Comment by postlund** [comment]
Can you measure the time until it happens? My guess is that it happens faster circa 15 seconds. I can see that I have put a wait_for call with that as timeout. Might be a good idea to remove that since it causes problems with longer running tasks.

**Comment by nriley** [comment]
It's exactly 15 seconds, the 1 vs 2 poll thing seems like just a coincidence.

```
03/18 14:13:37 Press ENTER to stop
03/18 14:13:37 Media type: Unknown
03/18 14:13:37 Play state: No media
03/18 14:13:37    Shuffle: False
03/18 14:13:37 --------------------
03/18 14:13:52 Traceback (most recent call last):
03/18 14:13:52   File "/Users/nicholas/Documents/Development/pyatv/src/pyatv/pyatv/__main__.py", line 306, in _run_application
03/18 14:13:52     return (yield from asyncio.wait_for(cli_handler(loop), timeout=15))
03/18 14:13:52   File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/asyncio/tasks.py", line 356, in wait_for
03/18 14:13:52     raise futures.TimeoutError()
03/18 14:13:52 concurrent.futures._base.TimeoutError
03/18 14:13:52 
03/18 14:13:52 >>> An error occurred, full stack trace above
```

**Comment by postlund** [comment]
Yeah, that was my guess too. But great, I'll push a fix for this and make sure that it's included in 0.3.0 which is planned next week when I get som time.
